assembly-extensions/build.gradle

```diff
@@ -25,6 +25,7 @@ artifacts {
 }
 
 task embeddedAssemblyJar(type: Copy, dependsOn: shadowJar) {
+  duplicatesStrategy = 'include'
```

Comment:
Is there any problem with duplicates?

Reply:
Not a problem -> Gradle just complained about the deprecation warning. But after your note, I had a second look and actually we were really copying the file again in case it was build before (we haven't cleaned the build dir)